internal/pkg/cli/app_show.go

```diff
@@ -122,6 +146,28 @@ func (o *showAppOpts) description() (*describe.App, error) {
 		return nil, fmt.Errorf("list jobs in application %s: %w", o.name, err)
 	}
 
+	wkldDeployedtoEnvs := make(map[string][]string)
+	ctx, cancelWait := context.WithTimeout(context.Background(), waitForStackTimeout)
+	defer cancelWait()
+	g, ctx := errgroup.WithContext(ctx)
+	defer ctx.Done()
```

Comment:
I think we can delete this 🤔, we don't use the derived context
```suggestion
	g, _ := errgroup.WithContext(ctx)
```